cost-analyzer/templates/cost-analyzer-smtp-configmap.yaml

```diff
@@ -0,0 +1,23 @@
+{{- if .Values.kubecostProductConfigs }}
+{{- if .Values.kubecostProductConfigs.smtp }}
+{{- if .Values.kubecostProductConfigs.smtp.enabled }}
+# If the smtp.config is not specified, the configmap will not be created
+{{- if .Values.kubecostProductConfigs.smtp.config }}
+# If the secretname is specified, the configmap will not be created
+{{- if not .Values.kubecostProductConfigs.smtp.secretname }}
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ default "smtp-configs" .Values.smtpConfigmapName }}
```

Comment:
`smtpConfigmapName` is not defined anywhere.

Reply:
I was following the example given by `productConfigmapName` which, interestingly, also isn't defined anywhere. I'll get both of those added.